packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -294,6 +294,12 @@ export class GCFBootstrapper {
         if (triggerType === TriggerType.UNKNOWN) {
           response.sendStatus(400);
           return;
+        } else if (triggerType === TriggerType.SCHEDULER) {
+          // TODO: currently we assume that scheduled events walk all repos
+          // managed by the client libraries team, it would be good to get more
+          // clever and instead pull up a list of repos we're installed on by
+          // installation ID:
```

Comment:
This comment is not correct, but it's moved from below. We do pull the list of repositories by installation id.